src/components/header/styles.tsx

```diff
@@ -47,12 +55,15 @@ export const contactUsPopoverImageStyles = {
 export const contactUsPopoverMainContainerStyles = {
     width: "500px",
     "@media screen and (max-width: 500px)": {
-        width: "90vw",
+        width: "85vw",
     },
 };
 export const contactUsPopoverContainerStyles = {
     zIndex: "0",
     top: "-13px",
+    "@media screen and (min-width: 375px) and (max-width: 500px)": {
+        left: "6px",
+    },
```

Comment:
using media queries like this is perfectly fine, However, Material-UI has other ways to handle media queries, consider them in the future. This can make your code cleaner and easier to maintain, especially if you have a lot of responsive styles.